ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/ValidatedCommandFactory.java

```diff
@@ -136,6 +136,11 @@ private static Command createForAlterSystemQuery(
     final String propertyName = alterSystemProperty.getPropertyName();
     final String propertyValue = alterSystemProperty.getPropertyValue();
 
+    // raise exception if feature flag is set
+    if (!context.getKsqlConfig().getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED)) {
+      throw new KsqlServerException("This feature is not yet enabled.");
```

Comment:
Could we be more specific in the error message, like "Cannot alter system configs when KSQL_SHARED_RUNTIME_ENABLED is turned off."